packages/ui/__tests__/Button.test.tsx

```diff
@@ -117,6 +118,40 @@ describe('Button', () => {
     })
   })
 
+  it('Renders disabled button with a disabled tooltip, correct tooltipVisible flag passed to TruncatedText', async () => {
+    const disabledTooltip = 'Disabled tooltip'
+
+    const wrapper = await mountAndCheckA11Y(
+      // div is required because `axe` cannot validate react fragments
+      <div>
+        <Button disabled disabledTooltip={disabledTooltip}>
+          {label}
+        </Button>
+      </div>,
+    )
+
+    expect(wrapper.find(TruncatedText).props()).toMatchObject({
+      tooltipVisible: false,
+    })
+  })
+
+  it('Renders disabled button with a disabled tooltip and disabledTooltip is invisible, correct tooltipVisible flag passed to TruncatedText', async () => {
```

Comment:
nit: This description is a bit hard to read :smile: 
How about something simple?
`Renders disabled button with invisible tooltip`

Reply:
Adjusted